js/src/dropdown.js

```diff
@@ -480,17 +478,18 @@ class Dropdown extends BaseComponent {
       return
     }
 
-    if (!isActive && (event.key === ARROW_UP_KEY || event.key === ARROW_DOWN_KEY)) {
-      getToggleButton().click()
+    if (event.key === ARROW_UP_KEY || event.key === ARROW_DOWN_KEY) {
+      if (!isActive) {
+        getToggleButton().click()
+      }
```

Comment:
I think you missed a return here or not?
```suggestion
return
      }
```

Reply:
Uhm nope. The click is opening the dropdown the following code will handle the arrow keys. If we return here you have to hit the arrow keys twice again.